docs/core/tutorials/cli-create-console-app.md

```diff
@@ -154,8 +166,11 @@ Single files are fine for simple one-off programs, but if you're building a more
 
 Once you're ready to distribute your app, use the [dotnet publish](../tools/dotnet-publish.md) command to generate the _publish_ folder at _bin\\debug\\netcoreapp3.1\\publish\\_ (use `/` for non-Windows systems). You can distribute the contents of the _publish_ folder to other platforms as long as they've already installed the dotnet runtime.
 
-```console
+```dotnetcli
 dotnet publish
+```
+
+```console
 Microsoft (R) Build Engine version 16.4.0+e901037fe for .NET Core
 Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
I splitted into 2 snippets, the command in one snippet with dotnetcli identifier, and the output in another snippet with console id. Feel free if you want me to revert this.

Reply:
What's the reason to split it out? It seems a bit weird to me.

Reply:
@gewarren, I did that because I wanted to use different identifiers here. I've added a text between the splitted snippets.